README.md

```diff
@@ -35,21 +35,21 @@ Build using cmake.
 
 | Dependency Name            | Description                                                             | Version          |
 | ------------------         |-----------------------------------------------------------------------  | -------          |
-| RPR_LOCATION               | Radeon Pro Render directory with include and lib dirs                   | 1.34.7 or higher |
+| RPR_LOCATION               | Radeon Pro Render directory with include and lib dirs                   | 1.34.9 or higher |
 
 ##### Radeon Image Filters
 
 | Dependency Name            | Description                                                             | Version          |
 | ------------------         |-----------------------------------------------------------------------  | -------          |
-| RIF_LOCATION               | Radeon Image Filter Library directory with include and lib dirs         | 1.4.2 or higher  |
+| RIF_LOCATION               | Radeon Image Filter Library directory with include and lib dirs         | 1.4.4 or higher  |
```

Comment:
I know about coming SDK repository changes. I'm updating this versions for the sake of correctness because that's the versions I was using lately